draft-irtf-cfrg-vdaf.md

```diff
@@ -3663,37 +3663,56 @@ way. The parameters for this circuit are summarized below.
 
 ### Prio3Sum
 
-| Parameter         | Value                                      |
-|:------------------|:-------------------------------------------|
-| `Valid`           | `Sum(Field128, bits)` (this section)       |
-| `Field`           | `Field128` ({{fields}})                    |
-| `PROOFS`          | `1`                                        |
-| `Xof`             | `XofTurboShake128` ({{xof-turboshake128}}) |
+| Parameter         | Value                                          |
+|:------------------|:-----------------------------------------------|
+| `Valid`           | `Sum(Field64, max_measurement)` (this section) |
+| `Field`           | `Field64` ({{fields}})                         |
+| `PROOFS`          | `1`                                            |
+| `Xof`             | `XofTurboShake128` ({{xof-turboshake128}})     |
 {: title="Parameters for Prio3Sum."}
 
 The next instance of Prio3 supports summing of integers in a pre-determined
-range. Each measurement is an integer in range `[0, 2^bits)`, where `bits` is an
-associated parameter.
+range. Each measurement is an integer in `range(max_measurement+1)`, where
+`max_measurement` is an associated parameter equal to the largest valid
+measurement.
+
+The range check is accomplished by encoding the measurement in a way that
+ensures it is in range, then comparing the reported measurement to the range
+checked measurement. Let
+
+* `bits = max_measurement.bit_length()`, the number of bits needed to encode
+  the largest valid measurement
+
+* `offset = 2**bits - 1 - max_measurement`
+
+The range checked measurement is the bit-encoding of `offset` plus the
+measurement. Observe that only measurements between `0` and `max_measurement`
+inclusive can be encoded this way with `bits` bits.
```

Comment:
Same as below. My two main issues with the existing explanation are that it elides that the "reported measurement" is encoded as a bit vector as well, and the claim that the 'range checked measurement' ensures the measurement is "between `0` and `max_measurement`" is too strong, since that could represent a measurement of `-offset` as well.

```suggestion
The range check is accomplished by encoding the measurement as a bit vector,
encoding the measurement plus an offset as a bit vector, then checking that the
two encoded integers are consistent. Let

* `bits = max_measurement.bit_length()`, the number of bits needed to encode
  the largest valid measurement

* `offset = 2**bits - 1 - max_measurement`

The first bit-encoded integer is the measurement itself. Note that only
measurements between `0` and `2**bits - 1` can be encoded this way with
`bits` bits. The second bit-encoded integer is the sum of the measurement
and `offset`. Observe that only measurements between `-offset` and
`max_measurement` inclusive can be encoded this way with `bits` bits.
```